cmd/serverless-init/log/log_test.go

```diff
@@ -37,3 +37,12 @@ func TestIsEnabledFalse(t *testing.T) {
 	assert.False(t, isEnabled("1"))
 	assert.False(t, isEnabled("FALSE"))
 }
+
+func TestIsInstanceTailingEnabled(t *testing.T) {
+	t.Setenv("DD_AAS_INSTANCE_LOGGING_ENABLED", "false")
+	assert.False(t, isInstanceTailingEnabled())
+	t.Setenv("DD_AAS_INSTANCE_LOGGING_ENABLED", "True")
+	assert.True(t, isInstanceTailingEnabled())
+	t.Setenv("DD_AAS_INSTANCE_LOGGING_ENABLED", "")
+	assert.True(t, isInstanceTailingEnabled())
```

Comment:
Should `DD_AAS_INSTANCE_LOGGING_ENABLED` be False be default? I thought this was something we would give to customers if they were having issues with instance specific logging but would be off by default.

Reply:
We can change it, but we discussed releasing this as V2 of serverless-init. If we want to make `false` the default we won't need to do that.